packages/white-whale-std/src/pool_manager.rs

```diff
@@ -112,6 +112,16 @@ pub struct PairInfo {
 }
 impl PairInfo {}
 
+#[cw_serde]
+pub struct Config {
+    pub whale_lair_addr: Addr,
+    pub owner: Addr,
```

Comment:
Like you suggested, since we are using `cw_ownable` i don't think we should store the `owner` in the config as it's already being done on the `OWNERSHIP` item.